joomla/joomla-cms
# Group toolbar buttons for newsfeeds

### Summary of Changes
This PR continues the grouping of action buttons in the backend (see #22832, #22393, #23337 and #23336, #23374, #23368, #23380). This time the reditect links

### Testing Instructions
Go to the newsfeeds in the backend and check the toolbar. There should be a "Change status" dropdown now which activates when you select a redirect link.

### Additional Information
Performing "empty trash" is not possible: https://issues.joomla.org/tracker/joomla-cms/23403
